Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -188,40 +190,73 @@ void NANDImporter::ProcessFile(const NANDFSTEntry& entry, const std::string& par
   }
 }
 
-void NANDImporter::ExtractCertificates(const std::string& nand_root)
+bool NANDImporter::ExtractCertificates(const std::string& nand_root)
 {
-  const std::string title_contents_path =
-      nand_root + "/title/00000001/0000000d/content/00000011.app";
-  File::IOFile file(title_contents_path, "rb");
-  if (!file)
+  const std::string content_dir = nand_root + "/title/00000001/0000000d/content/";
+
+  File::IOFile tmd_file(content_dir + "title.tmd", "rb");
+  std::vector<u8> tmd_bytes(tmd_file.GetSize());
+  if (!tmd_file.ReadBytes(tmd_bytes.data(), tmd_bytes.size()))
   {
-    PanicAlertT("Unable to open %s! Refer to "
-                "https://dolphin-emu.org/docs/guides/wii-network-guide/ to set up "
-                "certificates.",
-                title_contents_path.c_str());
-    return;
+    ERROR_LOG(DISCIO, "ExtractCertificates: Could not read IOS13 TMD");
+    return false;
+  }
+
+  IOS::ES::TMDReader tmd(std::move(tmd_bytes));
+  IOS::ES::Content content_metadata;
+  if (!tmd.GetContent(1, &content_metadata))
```

Comment:
The IOS kernel is not guaranteed to have content index 1. You should use GetBootIndex to get the boot content index instead.